pkg/apis/core/types_common.go

```diff
@@ -25,9 +25,9 @@ const (
 	// ErrorInfraUnauthorized indicates that the last error occurred due to invalid infrastructure credentials.
 	// It is classified as a non-retryable error code.
 	ErrorInfraUnauthorized ErrorCode = "ERR_INFRA_UNAUTHORIZED"
-	// ErrorInfraInsufficientPrivileges indicates that the last error occurred due to insufficient infrastructure privileges.
+	// ErrorInfraUnauthenticated indicates that the last error occurred due to invalid authentication details.
```

Comment:
any changes needed to the doc string?

Reply:
actually I personally like a definition close to the HTTP error code. 
Unauthenticated -> https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401
Unauthorized -> https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403
That is my personal preference. Generally the doc strings look ok.